src/components/DisplaySetting.js

```diff
@@ -20,16 +17,18 @@ function DisplaySetting(props) {
 
     console.log("value=", value);
     console.log("name=", name);
+    console.log("in DisplaySetting tempAttributesArray=", tempAttributes);
     console.log("in DisplaySetting attributesToShow=", attributesToShow);
-    //   console.log("displaySettingComponent is=", displaySettingComponent);
+    setTempAttributes(
```

Comment:
This tempAttribute is having the attributes at one step slower, so the changes are showing after one step